pkg/cli/info/configmap_test.go

```diff
@@ -0,0 +1,195 @@
+package info
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/openshift-pipelines/pipelines-as-code/pkg/params"
+	"github.com/openshift-pipelines/pipelines-as-code/pkg/params/clients"
+	"github.com/openshift-pipelines/pipelines-as-code/pkg/pipelineascode"
+	testclient "github.com/openshift-pipelines/pipelines-as-code/pkg/test/clients"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	rtesting "knative.dev/pkg/reconciler/testing"
+)
+
+func TestIsGithubAppInstalled(t *testing.T) {
+	namespace := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "namespace1",
+		},
+	}
+
+	secret := &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      pipelineascode.DefaultPipelinesAscodeSecretName,
+			Namespace: namespace.GetName(),
+		},
+		Data: map[string][]byte{
+			"hub.token":      []byte(`Yzg5NzhlYmNkNTQwNzYzN2E2ZGExYzhkMTc4NjU0MjY3ZmQ2NmMeZg==`),
+			"webhook.secret": []byte(`NTA3MDc=`),
+		},
+	}
+	tests := []struct {
+		name       string
+		namespaces []*corev1.Namespace
+		secrets    []*corev1.Secret
+		want       bool
+	}{{
+		name:       "GithubApp is installed",
+		namespaces: []*corev1.Namespace{namespace},
+		secrets:    []*corev1.Secret{secret},
+		want:       true,
+	}, {
+		name: "GithubApp is not installed",
+		want: false,
+	}}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			ctx, _ := rtesting.SetupFakeContext(t)
+			tdata := testclient.Data{
+				Secret:     tt.secrets,
+				Namespaces: tt.namespaces,
+			}
+			stdata, _ := testclient.SeedTestData(t, ctx, tdata)
+			cs := &params.Run{
+				Clients: clients.Clients{
+					Kube: stdata.Kube,
+				},
+			}
+			out := IsGithubAppInstalled(ctx, cs, namespace.GetName())
+			if res := cmp.Diff(out, tt.want); res != "" {
+				t.Errorf("Diff %s:", res)
+			}
+		})
+	}
+}
+
+func TestGetPACInfo(t *testing.T) {
+	namespace := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "namespace1",
+		},
+	}
+
+	configMap := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      infoConfigMap,
+			Namespace: namespace.GetName(),
+			Labels: map[string]string{
+				"app.kubernetes.io/part-of": "pipelines-as-code",
+			},
+		},
+		Data: map[string]string{
+			"provider":       "GithubApp",
+			"controller-url": "https://smee.io/WKR2cP3ug5K6A92T",
```

Comment:
maybe use placeholder in code, instead of real url....

Reply:
updated to `"https://controller.url"`